page_analyzer/templates/index.html

```diff
@@ -1,58 +1,26 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title>Аналиратор страниц</title>
-    <link
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
-      rel="stylesheet"
-      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
-      crossorigin="anonymous"
-    />
-  </head>
-  <body class="min-vh-100 d-flex flex-column">
-    <header class="flex-shrink-0">
-      <nav class="navbar navbar-expand-md navbar-dark bg-dark px-3">
-        <a class="navbar-brand" href="/">Анализатор страниц</a>
-        <div id="navbarNav">
-          <ul class="navbar-nav">
-            <li class="nav-item"><a class="nav-link" href="">Сайты</a></li>
-          </ul>
-        </div>
-      </nav>
-    </header>
-    <div
-      class="col-12 col-md-10 col-lg-8 mx-auto border rounded-3 bg-light p-5"
+{% extends "base.html" %} {% block main %}
```

Comment:
same, let's put `block` onto a separate line

Reply:
the same